stripe.go

```diff
@@ -35,6 +35,13 @@ const (
 	// APIURL is the URL of the API service backend.
 	APIURL string = "https://api.stripe.com"
 
+	// ConnectURL is the URL for OAuth.
+	ConnectURL string = "https://connect.stripe.com"
```

Comment:
So now that I look at this again, I think the duplicate constant between `APIURL` and `apiURL` was basically a refactoring mistake, and really doesn't help anything particularly. I think it's somewhat convenient that these URLs are exported in case users want them, so would you mind just dropping `connectURL` in favor `ConnectURL`?